plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/resource/VmwareResource.java

```diff
@@ -4763,7 +4763,13 @@ protected Answer execute(MigrateCommand cmd) {
         final String vmName = cmd.getVmName();
         try {
             VmwareHypervisorHost hyperHost = getHyperHost(getServiceContext());
+            if (hyperHost == null) {
+                throw new CloudRuntimeException("no hypervisor found for migrate command");
```

Comment:
```suggestion
                throw new CloudRuntimeException("no hyperHost found for migrate command");
```

Reply:
i don t like your sugestion @vishesh92 , 'hyperhost' is not a valid english word. How about 
```suggestion
                throw new CloudRuntimeException("no hypervisor host found for migrate command");
```